src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -102,19 +105,17 @@ public void deleteMessageWithID(String messageID) {
    * Gets messages posted by a specific user.
    *
    * @return a list of messages posted by user, or empty list if no messages posted to country page.
-   *     List is sorted by time descending.
+   *         List is sorted by time descending.
    */
   public List<Message> getMessagesByUser(String user) {
-    Query query =
-        new Query("Message")
-            .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
-            .addSort("timestamp", SortDirection.DESCENDING);
+    Query query = new Query("Message")
+        .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
+        .addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
There's still a weird change here that Aaron had requested removed, I think you need to use 
`mvn com.coveo:fmt-maven-plugin:format` and then push the change because for some reason it didn't seem to take affect